core/src/main/java/com/mapzen/android/graphics/model/BitmapMarkerFactory.java

```diff
@@ -0,0 +1,21 @@
+package com.mapzen.android.graphics.model;
+
+import com.mapzen.android.graphics.internal.StyleStringGenerator;
+
+/**
+ * Creates {@link BitmapMarker} objects.
+ */
+public class BitmapMarkerFactory {
```

Comment:
I might not be following the usage of the Factory here, but could this functionality not make sense in `MarkerManager` itself?

Reply:
It definitely could be within `MarkerManager` but the reason I pulled it out was to make testing easier. If I have a separate object in charge of creating the markers that I inject into `BitmapMarkerFactory`, I can easily have this factory object generate mocks and make assertions on them when testing:
https://github.com/mapzen/android/pull/433/files#diff-f00bf2e8f7b3f14b7d6bc8fbf88ddabfR40
https://github.com/mapzen/android/pull/433/files#diff-f00bf2e8f7b3f14b7d6bc8fbf88ddabfR57

Reply:
ohh yes definitely makes sense. 👍